modules/cnf-disabling-cpu-cfs-quota.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+[id="cnf-disabling-cpu-cfs-quota_{context}"]
+= Disabling CPU CFS quota
+
+To eliminate CPU throttling for pinned pods, create a pod with the `cpu-quota.crio.io: "disable"` annotation. This annotation disables the CPU completely fair scheduler (CFS) quota when the pod runs.
+
+.Example pod specification with `cpu-quota.crio.io` disabled
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+      cpu-quota.crio.io: "disable"
+spec:
+    runtimeClassName: performance-<profile_name>
+...
```

Comment:
"For [snippets or sections of a file... for YAML](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks), use ('#…​')" throughout this code block

```suggestion
# ...
```